### PR TITLE
orchestrator-process: Fix context for reading secret

### PR DESCRIPTION
**Pull request by def-**
Follow-up to https://github.com/MaterializeInc/materialize/pull/28812

### Checklist

- [ ] This PR has adequate test coverage / QA involvement has been duly considered. ([trigger-ci for additional test/nightly runs](https://trigger-ci.dev.materialize.com/))
- [ ] This PR has an associated up-to-date [design doc](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/README.md), is a design doc ([template](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/00000000_template.md)), or is sufficiently small to not require a design.
  
- [ ] If this PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way), then it is tagged with a `T-proto` label.
- [ ] If this PR will require changes to cloud orchestration or tests, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
  
- [ ] If this PR includes major [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note), I have pinged the relevant PM to schedule a changelog post.

### CODE REVIEW

File: misc/python/materialize/mzcompose/services/materialized.py [hunk 1]
```diff
@@ -94,6 +94,13 @@ def __init__(
             *DEFAULT_CRDB_ENVIRONMENT,
         ]
 
+        if image == "materialize/materialized:v0.110.3":
+            print("HERE!")
+            image = "materialize/materialized:devel-abf064c9dd95566ee13f8053c1b287f9f220d591"
+        if image == "materialize/materialized:v0.109.1":
+            print("THERE!")
+            image = "materialize/materialized:devel-f16c0c1242775d8a579cc68684ef01f3b8f80101"
+
```
Comment (benesch) [inline misc/python/materialize/mzcompose/services/materialized.py:103]: I assume you didn't mean to include this here!

File: src/orchestrator-process/src/secrets.rs [hunk 2]
```diff
@@ -87,7 +87,7 @@ impl SecretsReader for ProcessSecretsReader {
     async fn read(&self, id: GlobalId) -> Result<Vec<u8>, anyhow::Error> {
         let contents = fs::read(self.secrets_dir.join(id.to_string()))
             .await
-            .with_context(|| "reading secret {id}")?;
+            .with_context(|| format!("reading secret {id}"))?;
```
Comment (benesch) [inline src/orchestrator-process/src/secrets.rs:90]: Ugh! Thanks for catching this.

**Review by benesch — APPROVED** [review]
Thank you for catching this!

File: misc/python/materialize/mzcompose/services/materialized.py (hunk 1 above)
Comment (benesch) [reply misc/python/materialize/mzcompose/services/materialized.py]: Removed this and marked the updated PR for automerge.

Comment (def-) [reply misc/python/materialize/mzcompose/services/materialized.py]: Oops, of course. Thank you.

**Pull request merged**